DigitalOffice/Tests/UserServiceTests/CommandsTests/GetUserByIdCommandTests.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using NUnit.Framework;
+using Moq;
+using UserService.Commands;
+using UserService.Commands.Interfaces;
+using UserService.Database.Entities;
+using UserService.Mappers.Interfaces;
+using UserService.Models;
+using UserService.Repositories.Interfaces;
+
+namespace UserServiceUnitTests.CommandsTests
```

Comment:
rename folder to Commands (without Tests)
change namespace UserServiceUnitTests.Commands

Reply:
fixed